ExomeDepth/igv_xml_session.py

```diff
@@ -205,6 +101,14 @@ def make_single_igv_session(args, igv_extension, vcf_extension):
         elif args.pipeline == "nf":  # For NF pipeline.
             bam = "{0}.bam".format(args.sampleid.split(",")[0])  # Bam file
 
+    if args.reanalysis:
+        reanalysis = make_reanalysis(args.reanalysis)
+        igv_extension = "{}_ref.igv".format(settings.reanalysis_dic[reanalysis[args.sampleid]['suffix']][1])
+        vcf_extension = "exome_calls_{}.vcf".format(settings.reanalysis_dic[reanalysis[args.sampleid]['suffix']][1])
```

Comment:
Why do you add suffix as a prefix for the igv file and use it as a suffix for the vcf file?

Reply:
Renamed parameter too avoid confusion (by someone...)